Dockerfile

```diff
@@ -0,0 +1,41 @@
+# Use an official centos image as a parent image
+FROM centos:7
+
+# Set the working directory to install dependencies to /quattor
+WORKDIR /quattor
+
+# install library core in /quattor, tests need it
+ADD https://codeload.github.com/quattor/template-library-core/tar.gz/master /quattor/template-library-core-master.tar.gz
+RUN tar xvfz template-library-core-master.tar.gz
+
+# Copy the current directory contents into the container at /ncm-metaconfig
+#TODO: use docker volumes here to mount the current directory contents, no no need to rebuild every time to test
+ADD . /ncm-metaconfig/
```

Comment:
this needs more comment why this works (ie you are expected to run this from the ncm-metaconfig dir of the clone you are developing in)

Reply:
indeed, I'll mark this as WIP until I fix the todo and document some more